release_docs/CHANGELOG.md

```diff
@@ -443,8 +443,21 @@ Simple example programs showing how to use complex number datatypes have been ad
 
 ## Parallel Library
 
+  Subfiling VFD:
+
+  - Added H5FDsubfiling_get_file_mapping() API function to retrieve
+    the names of all physical subfiles that collectively make up a
+    logical HDF5 file when using the subfiling Virtual File Driver.
```

Comment:
I suspect you want these blurbs to use `###` markup for the title and the body be a single line with no `-` like the other entries.